pkg/rewards/claim.go

```diff
@@ -233,24 +239,24 @@ func Claim(cCtx *cli.Context, p utils.Prompter) error {
 				logger.Infof("Claim written to file: %s", config.Output)
 			} else {
 				fmt.Println(string(jsonData))
-				fmt.Println()
-				fmt.Println("To write to a file, use the --output flag")
+				logger.Info("To write to a file, use the --output flag")
 			}
 		} else {
 			if !common.IsEmptyString(config.Output) {
-				fmt.Println("output file not supported for pretty output type")
-				fmt.Println()
+				logger.Info("output file not supported for pretty output type")
 			}
 			solidityClaim := claimgen.FormatProofForSolidity(accounts.Root(), claim)
-			fmt.Println("------- Claim generated -------")
+			logger.Info("------- Claim generated -------")
 			common.PrettyPrintStruct(*solidityClaim)
-			fmt.Println("-------------------------------")
-			fmt.Println("To write to a file, use the --output flag")
+			logger.Info("-------------------------------")
+			logger.Info("To write to a file, use the --output flag")
+		}
+		if !cCtx.Bool(SilentFlag.Name) {
```

Comment:
you can also move this reading logic to the function which reads all the flags and populates the config so as to put all reading logic there